pkgs/tools/typesetting/tex/mftrace/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, makeWrapper
+, autoreconfHook
+, buildEnv
+, python3
+, fontforge
+, potrace
+, texlive
+}:
+
+# to use with a texlive distribution, ensure that the desired fonts and
+# the packages kpathsea, t1utils, metafont are available at runtime
+#
+# mftrace can also be added to nixpkgs' texlive via texlive.combine
+#
+# accepted overrides:
+# - potrace = autotrace
+# - fontforge = ghostscript (limited functionality)
+# - fontforge = null (limited functionality)
+
+let self = stdenv.mkDerivation rec {
+  pname = "mftrace";
+  version = "1.2.20";
+
+  # https://lilypond.org/download/sources/mftrace/mftrace-1.2.20.tar.gz
+  # is incomplete, fetch repo and use autoconf instead
+  # see https://github.com/hanwen/mftrace/issues/13
+  src = fetchFromGitHub {
+    owner = "hanwen";
+    repo = "mftrace";
+    rev = "release/${version}";
+    sha256 = "02ik25aczkbi10jrjlnxby3fmixxrwm2k5r4fkfif3bjfym7nqbc";
+  };
+
+  nativeBuildInputs = [ makeWrapper autoreconfHook python3 potrace ];
+
+  buildInputs = [ fontforge potrace ];
+
+  postInstall = ''
+    wrapProgram $out/bin/mftrace --prefix PATH : ${lib.makeBinPath buildInputs}
+  '';
+
+  passthru.tlType = "bin";
+  passthru.pkgs = [ self ] ++ (builtins.concatMap (builtins.getAttr "pkgs")
+    (with texlive; [ kpathsea t1utils metafont ]));
+
+  meta = with lib; {
+    description = "Scalable PostScript Fonts for MetaFont";
+    longDescription = ''
+      mftrace is a small Python program that lets you trace a TeX bitmap
+      font into a PFA or PFB font (A PostScript Type1 Scalable Font) or
+      TTF (TrueType) font.
+    '';
+    homepage = "https://lilypond.org/mftrace/";
+    license = with licenses; [ gpl2Only free ];
```

Comment:
```suggestion
    license = with licenses; [ gpl2Only mit ];
```

Reply:
Good catch. For the record, texlive lists xdvi (the origin of the piece of code in mftrace) as "Free license not otherwise listed", but indeed the text matches MIT word for word except for "the authors". I didn't know that.